Hunting Queries/AWSCloudTrail/AWS_Unused_UnsupportedCloudRegions.yaml

```diff
@@ -0,0 +1,38 @@
+id: e0d57543-acbd-428b-bb96-24a67506f84d
+name: Unused or Unsupported Cloud Regions
+description: |
+  'Adversaries may create cloud instances in unused geographic service regions in order to evade detection. 
+  Access is usually obtained through compromising accounts used to manage cloud infrastructure.
+  Refer: https://attack.mitre.org/techniques/T1535/'
+requiredDataConnectors:
+  - connectorId: AWS
+    dataTypes:
+      - AWSCloudTrail
+tactics:
+  - DefenseEvasion
+relevantTechniques:
+  - T1535
+query: |
+
+  let starttime = 14d;
+  let midtime = 2d;
+  let endtime = 1d;
+  // Generating historical table of all events per AccountId and Region
+  AWSCloudTrail
```

Comment:
lets look at using materialize() here as we are re-using the data and looking back more than a day or 2.  This will help with performance.